examples/todo-list/src/repositories/todo-list.repository.ts

```diff
@@ -21,17 +23,27 @@ export class TodoListRepository extends DefaultCrudRepository<
     Todo,
     typeof TodoList.prototype.id
   >;
+  public readonly author: HasOneRepositoryFactory<
+    Author,
+    typeof TodoList.prototype.id
+  >;
 
   constructor(
     @inject('datasources.db') dataSource: juggler.DataSource,
-    @repository.getter(TodoRepository)
+    @repository.getter('TodoRepository')
     protected todoRepositoryGetter: Getter<TodoRepository>,
+    @repository.getter('AuthorRepository')
+    protected todoListImageRepositoryGetter: Getter<AuthorRepository>,
   ) {
     super(TodoList, dataSource);
     this.todos = this._createHasManyRepositoryFactoryFor(
       'todos',
       todoRepositoryGetter,
     );
+    this.author = this._createHasOneRepositoryFactoryFor(
+      'author',
+      todoListImageRepositoryGetter,
```

Comment:
`todoListImageRepositoryGetter` is inconsistent with the target model name `Author`. Did you mean `authorRepositoryGetter` instead?